config/locales/en.yml

```diff
@@ -532,6 +532,12 @@ en:
         one: You have one request to upgrade a trial group
         other: You have %{count} requests to upgrade a trial group
       upgrade_requests_title: Upgrade requests
+    move:
+      organisation_search:
+        hint: Change which organisation this group belongs to
+        label: Move "%{group_name}" to another organisation
+        prompt: Select an organisation
+      success_message: Group's organisation has changed to %{org_name}
```

Comment:
There's an existing `groups.success_messages` key, it might be more consistent to add this under that as `groups.success_messages.move` instead